scripts/purge_queue.py

```diff
@@ -0,0 +1,109 @@
+import sys
+
+import argparse
+import boto3
+import json
+import more_itertools
+from azul import config
+import logging
+
+from urllib.parse import urlparse
+
+logger = logging.getLogger(__name__)
+parser = argparse.ArgumentParser(description=__doc__)
+
+subparsers = parser.add_subparsers(help="sub-command help", dest="command")
+
+parser_dump = subparsers.add_parser("dump", help="Dump contents of queue into designated file")
+parser_dump.add_argument("queue_name", help="Name of the queue to obtain the messages from")
+parser_dump.add_argument("file_name", help="Name of file to write output to")
+parser_dump.add_argument("--delete", "-d", action='store_true',
+                         help="Purge messages from the queue after writing them to the designated file. "
+                              "By default the messages will be returned to the queue")
+parser_dump.add_argument("--json-body", "-J", action="store_true", help="Serialize json for queue message body")
+parser_list = subparsers.add_parser("list", help="List all available queues")
+
+sqs = boto3.resource("sqs")
```

Comment:
Avoid allocating resources (any resources, not just boto3 resources) at module scope. Importing any module, even a script module should not have side-effects.